national-resources/netherlands.md

```diff
@@ -0,0 +1,213 @@
+--- 
+title: Netherlands
+country_code: NL
+description: this page discusses infectious disease research in the Netherlands
+  and provides information on relevant health authorities and initiatives
+contributors:
+  - Robin Navest
+  - Julia Lischke
+  - Jan-Willem Boiten
+  - Jeroen Belien
+  - Clara Amid
+  - Angelica Maineri
+  - Ricarda Braukmann
+coordinators:
+  - Robin Navest
+training:
+  - name: Workshops on delivering FAIR metadata for COVID-19 data portal
+    registry: Health-RI
+    url: https://www.health-ri.nl/initiatives/dutch-covid-19-data-support-programme/workshops-delivering-fair-metadata-covid-19-data
+national_resources:
+  - name: COVID-NL clinical data dashboard
+    description: The Dutch national COVID-19 clinical data dashboard allows
+      exploration and reuse of clinical data from Dutch university medical
+      centers (UMCs). The dashboard provides researchers with a clear overview
+      of what is available, allows searching for specific data and makes access
+      to such data easier when the necessary ethical and legal conditions have
+      been met.
+    how_to_access: request data through the dashboard
```

Comment:
```suggestion
    how_to_access: #request data through the dashboard
```
Commenting out. The `how_to_access` field is generally used to explain access to the resource itself when it is closed/restricted access. But not to the data inside the resource.
I am copy/pasting the same comment to the other

Reply:
Thanks for the explanation. I've changed it to "the policy for access to and sharing of clinical COVID-19 is described in https://elsi.health-ri.nl/sites/elsi/files/2022-03/HRI_COVID-NL%20policy%20doc%20v2.0%20final.pdf"